core/src/test/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerDAOSpec.scala

```diff
@@ -56,9 +51,9 @@ class BillingProfileManagerDAOSpec extends AnyFlatSpec with MockitoSugar {
   )
   val testContext = RawlsRequestContext(userInfo)
 
-  behavior of "listBillingProfiles"
+  behavior of "getAllBillingProfiles"
 
-  it should "return billing profiles to which the user has access" in {
+  it should "return all profiles from listBillingProfiles when the profiles exceeds the request batch size" in {
```

Comment:
This diff is a bit confusing, but the this is the result of deleting the `return billing profiles to which the user has access` test; I believe it was really only exercising that we were getting the hardcoded billing profile back and the `return all profiles from listBillingProfiles when the profiles exceeds the request batch size` test exercises the listing functionality sufficiently.